src/earthengine/ee_worker.js

```diff
@@ -292,15 +281,22 @@ class EarthEngineWorker {
             useCentroid,
             style,
             tileScale = DEFAULT_TILE_SCALE,
+            unmaskAggregation,
         } = this.options
         const singleAggregation = !Array.isArray(aggregationType)
         const useHistogram =
             singleAggregation &&
             hasClasses(aggregationType) &&
             Array.isArray(style)
-        const image = await this.getImage()
         const scale = this.eeScale
         const collection = this.getFeatureCollection()
+        let image = await this.getImage()
+
+        // Used for "constrained" WorldPop layers
+        // We need to unmask the image to get the correct population density
+        if (unmaskAggregation) {
+            image = image.unmask(0)
```

Comment:
Maybe we want this to be a parameter instead of "0". There could be scenario where we want the default value to be something else.

Reply:
I have changed so "unmaskAggregation" can be a boolean or a number.